crates/uv/tests/common/mod.rs

```diff
@@ -44,12 +44,12 @@ macro_rules! deterministic_lock {
             // Run the first resolution.
             $($x)*
 
-            // Run a second resolution with the new lockfile.
-            $($x)*
-
-            // Run a final clean resolution without a lockfile to ensure identical results.
-            let _ = fs_err::remove_file(&$context.temp_dir.join("uv.lock"));
-            $($x)*
+            // // Run a second resolution with the new lockfile.
+            // $($x)*
+            //
+            // // Run a final clean resolution without a lockfile to ensure identical results.
+            // let _ = fs_err::remove_file(&$context.temp_dir.join("uv.lock"));
+            // $($x)*
```

Comment:
Should this be a STOPSHIP?

Reply:
I mean, going back to doing deterministic checking.

Reply:
I replaced all the usages of this in `lock` with a second call to `.lock` with `--locked`. The `deterministic!` macro just makes it really hard to update the snapshots since you have to manually edit the tests, and `--locked` will fail more aggressively since it doesn't _just_ require that the string contents changed.

(I've uncommented this though, and left it in the ecosystem checks.)

Reply:
Does removing the `with_duplicates` fix the snapshot updating issue?